Formula/metview.rb

```diff
@@ -41,21 +43,22 @@ class Metview < Formula
   end
 
   def install
-    ENV["RPC_PATH"] = HOMEBREW_PREFIX
-    cmake_rpc_flags = if OS.linux?
-      "-DCMAKE_CXX_FLAGS=-I#{HOMEBREW_PREFIX}/include/tirpc"
-    else
-      ""
-    end
-
-    args = %w[
+    args = %W[
       -DBUNDLE_SKIP_ECCODES=1
       -DENABLE_MIR_DOWNLOAD_MASKS=OFF
       -DENABLE_BUILD_TOOLS=OFF
       -DENABLE_ECKIT_CMD=OFF
-      -DFFTW_PATH=#{HOMEBREW_PREFIX}
+      -DFFTW_PATH=#{Formula["fftw"].opt_prefix}
     ]
-    system "cmake", "-S", ".", "-B", "build", *args, *cmake_rpc_flags, *std_cmake_args
+
+    if OS.linux?
+      args += [
+        "-DRPC_PATH=#{Formula["libtirpc"].opt_prefix}",
+        "-DRPC_INCLUDE_DIRS=#{Formula["libtirpc"].opt_include}/tirpc",
```

Comment:
Maybe:
```suggestion
        "-DRPC_INCLUDE_DIR=#{Formula["libtirpc"].opt_include}/tirpc",
```

CMake ends up doing `set( RPC_INCLUDE_DIRS ${RPC_INCLUDE_DIR} )`

Reply:
Nice, that worked. Thanks.